components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -276,6 +289,17 @@ void BraveReferralsService::OnReferralInitLoadComplete(
   const base::Value* download_id = root->FindKey("download_id");
   pref_service_->SetString(kReferralDownloadID, download_id->GetString());
 
+  // We have reported an installation to promo server. We can kill the timer
+  // and ensure we don't send this request anymore.
+  pref_service_->SetBoolean(kReferralReportedInstall, true);
+  if (installation_timer_) {
+    DCHECK(installation_timer_->IsRunning());
+    installation_timer_->Stop();
+    DCHECK(!installation_timer_->IsRunning());
+    installation_timer_.reset();
```

Comment:
Not really sure we need to reset the pointer, though I guess it doesn't hurt either.

Reply:
We want to free it in the normal case where the user fires up the browser and it initializes correctly. The memory allocated for the timer just sticks around... Also the docs in timer.h say the timer is stopped once it goes out of scope, so I was wondering if we could just called `reset()` here instead of `Stop()`.